.circleci/config.yml

```diff
@@ -222,3 +265,13 @@ workflows:
           filters:
             branches:
               only: dev
+  nightly-integration-tests:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only: dev
+    jobs:
+      - integration-tests-alpha
```

Comment:
TOL: A specific time is a good first pass. I think the alpha build is somewhat variable and sometimes run on-demand. I wonder if it would be possible to trigger CircleCI from successful completion of the alpha build in Jenkins... maybe something like Google Pub/Sub or Cloud Tasks.

Reply:
All good thoughts on timing; for the moment these are based on Gary's requests when I talked to him yesterday.